Signal/src/util/StringUtil.h

```diff
@@ -1,25 +0,0 @@
-//
-//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
-//
-
-@interface NSString (Util)
```

Comment:
There's a lot of stale code that we can remove.  I'm pulling out the more obvious chunks first.

Reply:
🎉